src/components/Button/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import styles from './Button.module.scss';
+import { Loader, LoaderSize } from '../Loader/Loader';
+
+export type ButtonProps = React.PropsWithChildren<{
+  loading?: boolean;
+  children?: React.ReactNode;
+  classname?: string;
+}> &
+  React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const Button: React.FC<ButtonProps> = ({
+  loading,
+  children,
+  className,
+  ...otherActions
+}) => {
+  return (
+    <button
+      className={`${styles.button} ${className} 
+        ${loading || otherActions.disabled ? `${styles.button__disabled}` : ''}
+        `}
```

Comment:
красивее будет использовать либу classnames и тогда написать типа:
```suggestion
      className={cn(
        styles.button,
        className,
        (loading || otherActions.disabled) && styles.button__disabled
      )}
```